index.js

```diff
@@ -83,16 +83,15 @@ app.get('/tweet/:tweetId/:address', (req, res) => {
     })
     .catch((err) => {
       console.log(err)
-      respond('catch-error')
+      respond('catch-error', 'catch-error')
```

Comment:
I added the copied parameter to make the test simpler. I would like to refactor this once  this endpoint is tested.

Reply:
I didn't because the contract expect a string only if the verification is successful. But at some point I added to the Store contract a check to require that the string contrains only digits, so it should be ok.
The check is at https://github.com/Tweedentity/store/blob/master/contracts/TweedentityStore.sol#L81